package.json

```diff
@@ -133,6 +134,7 @@
     "nock": "^10.0.6",
     "pkg": "^4.4.2",
     "prettier": "^2.5.1",
+    "regenerator-runtime": "^0.13.9",
```

Comment:
I think this probably needs to be in the dependencies section

Reply:
yes. moved them to dependencies!